packages/snaps-execution-environments/src/common/endowments/webhid.ts

```diff
@@ -0,0 +1,24 @@
+import { rootRealmGlobal } from '../globalObject';
+
+/**
+ * Create a {@link navigator} object, with the same properties as the global
+ * {@link navigator} object, but only with access to hid.
+ *
+ * @returns The {@link navigator} object with only access to hid.
+ */
+function createHID() {
```

Comment:
We need to have a discussion about this endowment in general since it is not available in Firefox. Any snap that tries to leverage it will not work in FF. In the past we have not added endowments that aren't platform agnostic.

Reply:
Perhaps the snap manifest can have a target field?

```json
{
  "targets": {
    "chrome": ">=89",
  },
}
```